tests/test_phot.py

```diff
@@ -0,0 +1,33 @@
+from astropy.io import fits
+
+from calcos import phot
+
+
+def test_do_phot():
+    # Setup
+    obsmode = ["cos,nuv,mirrora,psa", "cos,nuv,mirrora,boa", "cos,nuv,mirrorb,psa", "cos,nuv,mirrorb,boa"]
+    hdr = fits.open("51c1638pi_imp.fits")  # fits extension header (that will be updated)
+    # imphttab = hdr[1]  # imaging photometric parameters table
+    # Test
+    # todo use for loop
+    # todo test the values of
+    #  photflam, photfnu, photbw, photplam, photzpt for a change
+    phot.doPhot(obsmode[0], hdr)
+    # Verify
+
```

Comment:
We have decided to put this test on the backburner for now because it's not used in CalCOS from what we can tell. Let's remove this test from this branch and put it on a separate branch called

Reply:
This should still be put on a separate branch and removed from this one